src/AutomationTests/SnapshotCommandUnitTests.cs

```diff
@@ -251,41 +283,52 @@ public void Execute_NoErrors_NoOutputFiles()
             _scanToolsMock.VerifyAll();
             _nativeMethodsMock.VerifyAll();
             _targetElementLocatorMock.VerifyAll();
+            _outputFileHelperMock.VerifyAll();
             _actionsMock.VerifyAll();
             _resultsAssemblerMock.VerifyAll();
         }
 
         [TestMethod]
         [Timeout(1000)]
-        public void Execute_NullOutputFileHelper_ThrowsException()
+        public void Execute_NullOutputFilePath_ThrowsException()
```

Comment:
The diff here gets a little muddled because a method was moved. It's clearer to review the 2 separate commits:
[Commit 1 (removed test is commented out)](https://github.com/microsoft/axe-windows/pull/276/commits/c7efa3c438d6485afe7ad3b57a3ca269b571c63b)
[Commit 2 (commented out test is deleted)](https://github.com/microsoft/axe-windows/pull/276/commits/37c9ede56193b6211c0fee99cf891cd38f455374)